src/main/java/net/sf/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -1102,7 +1107,7 @@ public void stateChanged(ChangeEvent event) {
 
 
     class DeleteAction extends AbstractAction {
-        public DeleteAction() {
+        DeleteAction() {
```

Comment:
I think, checkstyle or codacy or some other toal moans because of package private. Nevertheless, I think, it is OK to reduce visibility.